### PR TITLE
Make kubectl gke jobs test against stable1, rather than pin to 1.7

### PR DESCRIPTION
**Pull request by krzyzacy**
gce jobs are already doing that, missed gke ones

stable1 is pointing at 1.8 now and will move to 1.9 shortly (once we kill 1.6), ref https://github.com/kubernetes/test-infra#release-branch-jobs--image-validation-jobs

/assign @crimsonfaith91 @mengqiy

### CODE REVIEW

File: testgrid/config/config.yaml [hunk 1]
```diff
@@ -3681,30 +3681,30 @@ dashboards:
...
+  - name: skew-cluster-stable1-kubectl-latest-gce-serial
     test_group_name: ci-kubernetes-e2e-gce-new-master-gci-kubectl-skew-serial
-    description: '1.7 serial tests run against a 1.7 gke cluster using a 1.8 kubectl binary'
-  - name: skew-cluster1.7-kubectl1.8-gke-serial
+    description: 'stable1 serial tests run against a stable1 gke cluster using a master kubectl binary'
+  - name: skew-cluster-stable1-kubectl-latest-gke-serial
     test_group_name: ci-kubernetes-e2e-gke-new-master-gci-kubectl-skew-serial
-    description: '1.7 serial tests run against a 1.7 gce cluster using a 1.8 kubectl binary'
-  - name: skew-cluster1.8-kubectl1.7-gce-serial
+    description: 'stable1 serial tests run against a stable1 gce cluster using a master kubectl binary'
```
Comment (crimsonfaith91) [inline testgrid/config/config.yaml]: gke cluster?

File: testgrid/config/config.yaml [hunk 2]
```diff
@@ -3681,30 +3681,30 @@ dashboards:
...
+  - name: skew-cluster-latest-kubectl-stable1-gke
     test_group_name: ci-kubernetes-e2e-gke-master-new-gci-kubectl-skew
-    description: '1.7 e2e tests run against a 1.8 gke cluster using a 1.7 kubectl binary'
-  - name: skew-cluster1.7-kubectl1.8-gce-serial
+    description: 'stable1 e2e tests run against a master gke cluster using a stable1 kubectl binary'
+  - name: skew-cluster-stable1-kubectl-latest-gce-serial
     test_group_name: ci-kubernetes-e2e-gce-new-master-gci-kubectl-skew-serial
-    description: '1.7 serial tests run against a 1.7 gke cluster using a 1.8 kubectl binary'
-  - name: skew-cluster1.7-kubectl1.8-gke-serial
+    description: 'stable1 serial tests run against a stable1 gke cluster using a master kubectl binary'
```
Comment (crimsonfaith91) [inline testgrid/config/config.yaml]: gce cluster?

Comment (krzyzacy) [reply testgrid/config/config.yaml]: good catch (it was wrong from the beginning)

**Comment by crimsonfaith91** [comment]
@krzyzacy Everything looks good to me. I will check the test grid when the PR has merged.

**Pull request merged**